src/bower_components/alameda/alameda.js

```diff
@@ -239,6 +239,7 @@ var requirejs, require, define;
         }
 
         function delayedError(e) {
+            console.log(e.stack);
```

Comment:
I'm guessing this was just added for debugging? Should probably remove it so we aren't modifying 3rd party libs.

Reply:
I left this one in intentionally, as without it you don't get the source of errors, just a cryptic line of text (see #306 for example)